crates/router/src/utils/user/theme.rs

```diff
@@ -0,0 +1,158 @@
+use std::path::PathBuf;
+
+use common_utils::{id_type, types::theme::ThemeLineage};
+use error_stack::ResultExt;
+use hyperswitch_domain_models::merchant_key_store::MerchantKeyStore;
+
+use crate::{
+    core::errors::{StorageErrorExt, UserErrors, UserResult},
+    routes::SessionState,
+};
+
+fn get_theme_dir_key(theme_id: &str) -> PathBuf {
+    ["themes", theme_id].iter().collect()
+}
+
+pub fn get_specific_file_key(theme_id: &str, file_name: &str) -> PathBuf {
+    let mut path = get_theme_dir_key(theme_id);
+    path.push(file_name);
+    path
+}
+
+pub fn get_theme_file_key(theme_id: &str) -> PathBuf {
+    get_specific_file_key(theme_id, "theme.json")
```

Comment:
use constants.

Reply:
These are used only one time. Will move them once we are using it one more time.